src/components/Admin/AdminComponents/CreateListing.js

```diff
@@ -37,6 +37,7 @@ const CreateListing = () => {
     // Function that changes the state of the overlay when button is clicked
     const handleClose = () => {
         setOpen(false);
+        window.location.href = "http://localhost:3000/admin/listing";
```

Comment:
can you create a constant in the the env variables (REACT_APP_URL) something along those lines and replace localhost:3000 with the constant? or instead, try use linking to /admin/listing (there is a way if you don't use http:// link, react knows to get the parent link and attach /whateverlink/route/ to the parent link)